.gitignore

```diff
@@ -1,8 +1,8 @@
 # See http://help.github.com/ignore-files/ for more about ignoring files.
 # compiled output
 /dist
-/cc_purkukartoitus_service/dist/
-/cc_purkukartoitus_service/node_modules/
+/cc-pk-service/dist/
```

Comment:
hmm, we could possibly ignore all dist folder with pattern matching, like **/dist and **/node_modules. also with .env = **/.env